packages/propel/src/toast/toast.stories.tsx

```
@@ -0,0 +1,353 @@
+            message: "Your changes have been saved successfully.",
+          })
+        }
+        className="rounded bg-green-500 px-4 py-2 text-white hover:bg-green-600"
+      >
+        Show Success Toast
+      </button>
+    </>
+  ),
+};
+
+export const Error: Story = {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)